portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>catyu-step-2020</deploy.projectId>
```

Comment:
nit: missed this before in a previous PR i guess but you can probably remove the TODO above since it's already done.